src/amo/components/CategoriesHeader/styles.scss

```diff
@@ -0,0 +1,38 @@
+@import "~amo/css/inc/vars";
+@import "~core/css/inc/mixins";
+@import "~ui/css/vars";
+
+.CategoriesHeader .Card-contents {
+  display: flex;
+}
+
+.CategoriesHeader-loadingText {
+  @include margin-end(5px);
+}
+
+.CategoriesHeader-list {
+  display: block;
+  list-style: none;
+  margin: 0 auto;
+  padding: 0;
```

Comment:
They're for `ul` and `li` items which have default styles. Not sure where we're declaring those but they do exist 😒